src/component/SignUpForm.js

```diff
@@ -0,0 +1,89 @@
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { useNavigate } from 'react-router-dom';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import { signUp } from '../redux/user/user';
+import '../css/SignUpForm.css';
+
+const SignUpForm = () => {
+  const dispatch = useDispatch();
+  const signUpAction = bindActionCreators(signUp, dispatch);
+  const navigate = useNavigate();
+
+  const [email, setEmail] = useState('');
+  const [userName, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+  const [passwordConfirmation, setPasswordConfirmation] = useState('');
+
+  return (
+    <div className="row bg-image-sign-up mx-0 d-flex flex-column justify-content-center align-items-center">
+      <div className="col-sm-6">
+        <h2 className="light-font">Sign Up</h2>
+        <Form
+          onSubmit={async (e) => {
+            e.preventDefault();
+            await signUpAction(userName, email, password, passwordConfirmation);
+            setEmail('');
+            setPassword('');
+            setUserName('');
+            setPasswordConfirmation('');
+            navigate('/');
+          }}
```

Comment:
- [ ] Again, this onSubmit code could be transported to a separate function outside of the return statement of the component, what do you think?